legate/core/launcher.py

```diff
@@ -747,7 +747,7 @@ def __del__(self) -> None:
     def add_scalar_arg(
         self,
         value: Any,
-        dtype: DTType,
+        dtype: Union[DTType, tuple[DTType, ...]],
```

Comment:
I think the current usage is more restricted than this; only singleton tuples are supported. Therefore, I think we could strengthen the type to `Union[DTType, tuple[DTType]]` -- assuming mypy is ok with that.

Reply:
Thanks for the clarification https://github.com/nv-legate/legate.core/pull/306/commits/97c35c87714929bf02c28b9f26901e5480b4cacf